Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
+
+# Install dependencies.
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     python python3-pip python3-tk texlive-latex-base latexmk git emacs locales
```

Comment:
Don't forget Vim.

Reply:
Are emacs or vim actually dependencies required for it to work?

Reply:
No, but you might need an editor if you are in the container.
```
vtomole@vtomole:~/Cirq$ docker run -it cirq/cirq bash
root@f8928a0fb7e4:/usr/src/app# ls
#README.rst#	 Dockerfile~  WORKSPACE			   check		   dev_tools		  python2.7-requirements.txt  setup.cfg
AUTHORS		 LICENSE      apt-system-requirements.txt  cirq			   docs			  release.md		      setup.py
CONTRIBUTING.md  MANIFEST.in  ascii.py			   cirq.egg-info	   examples		  requirements.txt	      style.md
Dockerfile	 README.rst   ascii.py~			   continuous-integration  python2.7-generate.sh  res.txt		      third_party
root@f8928a0fb7e4:/usr/src/app# vim README.rst 
bash: vim: command not found

```

Reply:
Done